### ISSUE TITLE
Log entries were created during Import when setup is not enabled

### ISSUE THREAD

**Issue by RadoArvay** [Drakonian/data-editor-for-bc#30]
My collegue has tried Import Table Data function - he used Excel format. He found out after import that the log entries were created even he did not enabled a logging option in the setup.

Could you please check it?

**Comment by Drakonian** [comment]
@RadoArvay 
Yes, there is indeed such a flaw, as soon as I fix it I will let you know.

**Comment by Drakonian** [comment]
Fixed in latest 3.0.0.7 version.
28463878c33d6079fdd4484c7327a9e4fe688cf0
